requirements/constraints.txt

```diff
@@ -1,5 +1,2 @@
-
 # Django LTS version
-django>=2.2,<2.3
-
-futures==3.1.1; python_version > "2.7"
+django>=3.2,<3.3
```

Comment:
shouldn't this be `Django<4.0`

Reply:
I used `django<3.3` to keep it similar to what we have in [common constraints](https://github.com/edx/edx-lint/blob/master/edx_lint/files/common_constraints.txt#L16).

Reply:
there's no Django 3.3, that's why it should be `<4.0`

Reply:
Updated the constraint.